### PR TITLE
Add summary strings to related posts, blacklist, and hold for moderation settings

### PR DESCRIPTION
**Pull request by tonyr59h**
Addresses #3682.

### CODE REVIEW

**Comment by daniloercoli** [comment]
@tonyr59h Would you mind to merge develop into this branch?

**Comment by tonyr59h** [comment]
All set. Thanks for the slew of reviews @daniloercoli :smile:

File: WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java [hunk 1]
```diff
@@ -338,6 +339,12 @@ public boolean getShowRelatedPostImages() {
         return mSettings.showRelatedPostImages;
     }
 
+    public @NonNull String getRelatedPostsDescription() {
+        if (mActivity == null) return "";
+        String desc = mActivity.getString(getShowRelatedPosts() ? R.string.on : R.string.off);
```
Comment (daniloercoli) [inline WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java]: `getShowRelatedPosts() ` sounds weird to me. Should it be `isShowRelatedPosts()` ?

**Comment by daniloercoli** [comment]
Seems OK to me. :shipit:

**Pull request merged**